eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/configuration/batch/io/RSMergeAndSplitCandidatesReaderConfiguration.java

```diff
@@ -75,12 +75,14 @@ public MongoDbCursorItemReader<SubmittedVariantOperationEntity> rsSplitCandidate
                                                                                             InputParameters parameters)
     {
         MongoDbCursorItemReader<SubmittedVariantOperationEntity> mongoItemReader = new MongoDbCursorItemReader<>();
-        mongoItemReader.setTemplate(mongoTemplate);
+        mongoItemReader.setMongoTemplate(mongoTemplate);
         mongoItemReader.setTargetType(SubmittedVariantOperationEntity.class);
         mongoItemReader.setCollection(SUBMITTED_VARIANT_OPERATIONS_COLLECTION);
 
         //See https://docs.google.com/spreadsheets/d/1KQLVCUy-vqXKgkCDt2czX6kuMfsjfCc9uBsS19MZ6dY/#rangeid=1213746442
         Meta meta = new Meta();
+        meta.setCursorBatchSize(parameters.getChunkSize());
+        meta.setMaxTime(Duration.ofHours(2));
```

Comment:
Had to look up the distinction between max processing time vs. cursor idle time...  should this duration be configurable or is that overkill?

Reply:
Sorry, this shouldn't have made it to the PR in the first place because the session idle timeout (which we are resetting using the periodic refresh routine) overrides this setting anyway. I initially added this when I was experimenting with every option under the sun.